salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/Hooks.java

```diff
@@ -15,6 +16,18 @@
  * [SL] This class contains hooks for the project.
  */
 public class Hooks {
+
+    private Context context;
```

Comment:
Why did you add the Context?? Is it necesary here?

Reply:
Ok, I delete these line unnecessaries